test/TestLedgerProcess.hs

```diff
@@ -21,29 +28,24 @@ import qualified Data.Expenses.Ledger.Process as LP
 |-}
 stripUnstableAttributes :: String -> String
 stripUnstableAttributes xml =
-  let
-    mtch = xml *=~ [re| (id|ref|version)="[^"]*">|]
-  in
-  replaceAll ">" mtch
+  let mtch = xml *=~ [re| (id|ref|version)="[^"]*">|] in replaceAll ">" mtch
 
 
 -- | Replace CRLF with LF.
 normalizeNewlines :: String -> String
-normalizeNewlines s =
-  let
-    mtch = s *=~ [re|\r\n|]
-  in
-  replaceAll "\n" mtch
+normalizeNewlines s = let mtch = s *=~ [re|\r\n|] in replaceAll "\n" mtch
 
 
 ledgerProcessSpec :: Spec
 ledgerProcessSpec =
-  describe "Data.Expenses.Ledger.Process" $
-    describe "xmlOfString" $
-      it "should return digits for simple input" $
-        stripUnstableAttributes . normalizeNewlines <$> LP.xmlOfString simpleLedgerJournal
-          `shouldReturn` simpleLedgerXml
-    where
+  describe "Data.Expenses.Ledger.Process"
+    $              describe "xmlOfString"
+    $              it "should return digits for simple input"
+    $              stripUnstableAttributes
+    .              normalizeNewlines
+    <$>            LP.xmlOfString simpleLedgerJournal
+    `shouldReturn` simpleLedgerXml
```

Comment:
I really don't like that `describe` and `it` have equal alignment here.

This makes sense from the perspective of the code formatter, but not from the perspective of
reading the code.